backend/remote-state/gcs/backend.go

```diff
@@ -162,22 +147,22 @@ func (b *Backend) configure(ctx context.Context) error {
 			return fmt.Errorf("Error parsing credentials '%s': %s", contents, err)
 		}
 
-		conf := jwt.Config{
```

Comment:
Why is removing this safe, exactly? I'm not 100% read up on auth details right now, so I could be missing something obvious here 🙂

Reply:
It was building the http client directly from the JWT of the service account which isn't great and `option.WithCredentialsJSON()` is simpler to use.

On a second look, I could have reused the `option.WithHTTPClient(conf.Client(ctx))` but I would prefer to avoid interacting with http.client unless it is necessary.